tutorials/xr/xr_full_screen_effects.rst

```diff
@@ -0,0 +1,81 @@
+.. _doc_xr_full_screen_effects:
+
+XR full screen effects
+======================
+
+When adding custom full screen effects to your XR application, one valid approach is
+applying a full screen quad to the XR camera and applying effects to that quad's shader.
+
+.. image:: img/xr_full_screen_effects_starting_quad.webp
+
+Here is a simple vignette shader that you might apply to the quad, darkening the edges
+of a user's screen:
+
+.. code-block:: glsl
+
+  shader_type spatial;
+  render_mode depth_test_disabled, skip_vertex_transform, unshaded, cull_disabled;
+
+  void vertex() {
+	  POSITION = vec4(VERTEX.xy, 1.0, 1.0);
+  }
+
+  void fragment() {
+	  ALBEDO = vec3(0.0);
+	  ALPHA = dot(UV * 2.0 - 1.0, UV * 2.0 - 1.0) * 2.0;
+  }
+
+
+However, when creating an effect that is centered straight ahead in the user's view
+(such as the above vignette effect), the end result may look incorrect in XR.
```

Comment:
This is shown wonderfully in the picture, but I think we should also say in words, why it doesn't look correct in XR (that the center of the rendered output we are producing isn't necessarily the center of the user's view in XR, because our output will be deformed to match the lenses of the headset)

Also, I think it would also be cool to show a screenshot from inside the headset showing how the correct and incorrect versions look, in addition to the screenshots of the render target texture. Maybe it could be shown in a 2x2 grid, with the headset image above the render target image? Otherwise, it might not be entirely clear that the off-center effect on the render target is actually the correct one that will look centered in the headset